### PR TITLE
Correct capitalization of an upper-cased string

### PR DESCRIPTION
**Pull request by morrido87**
Correctly capitalizes an uppercase string.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Contributions always welcome!
   - `isNegative()`
 - `isNumeric()`
 - ⭐ `isObject()` if it's a js plain Object
+- `isPastDate()`
```
Comment (ogroppo) [inline README.md:42]: should we have groups in the readme? otherwise it's on line 30

File: src/helpers/capitalize.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export const capitalize = (string: string) => {
-  return string.charAt(0).toUpperCase() + string.slice(1);
+  return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
```
Comment (ogroppo) [inline src/helpers/capitalize.ts:2]: cool, agreed!

**Review by ogroppo — APPROVED** [review]
just the readme thing

File: README.md (hunk 1 above)
Comment (morrido87) [reply README.md]: Oh, I didn't see it. Removed.

**Pull request merged**